examples/client.rs

```diff
@@ -35,10 +35,8 @@ use ring::rand::*;
 
 const MAX_DATAGRAM_SIZE: usize = 1350;
 
-const HTTP_REQ_STREAM_ID: u64 = 4;
-
 const USAGE: &str = "Usage:
-  client [options] URL
+  client [options] URL...
```

Comment:
While at it, do we want to add the `-n` option here as well? To make this more consistent with the http3-client.